### ISSUE TITLE
pmem-csi-periodic failure: imagefile: empty fsReadableForm

### ISSUE THREAD

**Issue by okartau** [intel/pmem-csi#654]
It is possible this is another side-effect of issue(s) already addressed in #647, but on first look it looks different than truncated string, so documenting  it here.
In the run 677 of pmem-csi-periodic, there is failure of imagefile test that seems like fsReadableForm had no value.
Failure:
```
check-imagefile.sh: SUCCESS: fstype= partition_size=534773760 block_size=4096
Error:          Not equal:
expected: "SUCCESS: fstype=ext2/ext3 partition_size=534773760 block_size=4096"
actual  : "SUCCESS: fstype= partition_size=534773760 block_size=4096"
```

Is it so that if fs was empty then Equal below has no chance to succeed, but we can detect that already before checking Equal, or even earlier, as empty fs value is not good to have already above,
If it remains not set because of communication error (?) then this can be detected earlier.
```
        fsReadableForm := fs // %T output from stat
        if fs == imagefile.Ext4 {
                // We don't bother with checking what is actually mounted,
                // this is close enough.
                fsReadableForm = "ext2/ext3"
        }
```

**Comment by okartau** [comment]
we dont have periodic job related issue about repeating failures, so commenting here. Good news is that periodic job had a clean run after long failing time, apparently merge of #647 improved it. Let's monitor for some time to determine was current issue covered by that as well.

**Comment by okartau** [comment]
series of recent periodic jobs clean, this really was a side-issue and got fixed via #647, closing
